governator-jetty/src/main/java/com/netflix/governator/guice/jetty/JettyModule.java

```diff
@@ -194,7 +206,7 @@ private Server getServer(OptionalJettyConfig optionalConfig, Set<JettyConnectorP
             webAppContext.setThrowUnavailableOnStartupException(true);
             webAppContext.addFilter(GuiceFilter.class, "/*", EnumSet.allOf(DispatcherType.class));
             webAppContext.addServlet(DefaultServlet.class, "/");
-            webAppContext.setResourceBase(config.getWebAppResourceBase());
+            webAppContext.setBaseResource(staticResourceBase);
```

Comment:
not equivalent to previous behavior; with this change, we're restricting to classpath-only.  Is that purposeful?

Reply:
This is the thing that I really wanted to understand a bit better. We get in this `if` block in the event that `staticResourceBase` is not null, but then that variable is never used anywhere. It seems like the intent would have been to use `staticResourceBase` here, but it's a bit unclear then what the `getWebAppResourceBase()` configuration was meant to control.

Personally, I just want to option to be able to use a classpath resource base instead of a file path resource base (which is what `setResourceBase` ends up giving). If you have some insight into how `staticResourceBase` and `webAppContextBase` are meant to interplay here maybe we could make that work in an intuitive way. But failing that, perhaps it should just be a new config flag whether we use `staticResourceBase` or `webAppContextBase`? Although in that case it's still not clear to be if this if block is checking the right thing.

Reply:
I just took a second pass at this. Rather than changing the behavior of staticResourceBase vs webAppResourceBase and disentangling what those mean, I made an update to the PR such that if webAppResourceBase starts with `classpath:` then it will treat it as a classpath resource. That should preserve backwards compatibility and be good enough for me.